security/cve_2011_4815_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../spec_helper', __FILE__)
+
+describe :resists_cve_2011_4815, shared: true do
+  it "resists CVE-2011-4815 by having different hash codes in different processes" do
+    eval("(#{@method}).hash.to_s").should_not == ruby_exe("print (#{@method}).hash")
+  end
+end
+
+describe "Object#hash" do
+  it_behaves_like :resists_cve_2011_4815, 'Object.new'
+end
+
+describe "Integer#hash" do
+  it_behaves_like :resists_cve_2011_4815, '14'
+end
+
+describe "Float#hash" do
+  it_behaves_like :resists_cve_2011_4815, '3.14'
+end
+
+describe "String#hash" do
+  it_behaves_like :resists_cve_2011_4815, '"abc"'
+end
+
+describe "Symbol#hash" do
```

Comment:
You can now use:
```ruby
  ruby_bug "#13376", "2.3"..."2.4" do
```
(thanks to https://github.com/ruby/mspec/commit/6856ff879835f2be337112fdf0645a192219efbc)